src/object.ts

```diff
@@ -18,9 +18,17 @@ namespace Shim {
 		return Object.getOwnPropertyNames(o).filter((key) => !Boolean(key.match(/^@@.+/)));
 	}
 
-	export function keys(o: any): string[] {
+	export function symbolAwareKeys(o: any): string[] {
```

Comment:
And like you said, hopefully nobody is exporting and using the `keys` method that was in the namespace...